app/src/app/app.module.ts

```diff
@@ -1,15 +1,19 @@
+import {HttpClientModule} from '@angular/common/http';
 import {NgModule} from '@angular/core';
+import {MatGridListModule} from '@angular/material';
```

Comment:
is this used anywhere?

Reply:
Ah no that's for another CL. Can remove for this one
Good catch! 🔥

Reply:
done.